Adding external facing workshop documentation (#755)
AlexsLemonade/training-modules: external-workshops/pre-workshop-meetings.md

# Pre-workshop meetings

<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*

* [Instructor onboarding meeting](#instructor-onboarding-meeting)
* [Participant set up meetings](#participant-set-up-meetings)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## Instructor onboarding meeting

> We strongly recommend that all external workshop instructors schedule an onboarding meeting after their workshop has been confirmed. 

Reviewer: ```suggestion
We strongly recommend that all external workshop instructors schedule an onboarding meeting with the Data Lab after their workshop has been confirmed. 
```

You will virtually meet with our team and ask any questions about our materials, getting set up, and instructing. 
If applicable, this is also a good time to get familiar with using our RStudio server, while we are there to help!  

[Please schedule a meeting using this link](https://meetings.hubspot.com/jen-omalley/external-workshop-onboarding-meeting). 
Remember, you can also reach out to us on [Slack](https://cancer-data-science.slack.com/) any time, and we will respond as soon as possible.

> ## Participant set up meetings

Reviewer: ```suggestion
## Participant setup meetings
```

> ### Pre-workshop set up meetings

Reviewer: ```suggestion
### Pre-workshop setup meetings
```

> **Optional, but recommended: To help your workshop run smoothly, we suggest offering pre-workshop set up meetings** to make sure participants and their computers are prepared ahead of time. 

Reviewer: ```suggestion
**Optional, but recommended: To help your workshop run smoothly, we suggest offering pre-workshop setup meetings** to make sure participants and their computers are prepared ahead of time. 
```

If you have the capacity, you can schedule brief individual meetings with participants. 
You can also consider holding open office hours during a video meeting or on Slack.

### Data pre-processing meetings

> **Required in some cases: We also advise holding data pre-processing meetings for participants who plan to work with their own data, need help removing personal health information (PHI) or personally identifiable information (PII), or are working with sequencing data from samples from human subjects before uploading it to a server.**

Reviewer: I'd split this up a little, maybe bullet points for which kinds of circumstances merit a meeting. Right now it's a big bold block.
